src/FSAL/FSAL_CORTXFS/handle.c

```diff
@@ -2005,6 +2115,7 @@ static fsal_status_t kvsfs_file_state_open(struct kvsfs_file_state *state,
 {
 	fsal_status_t status;
 
+	perfc_trace_inii(PFT_KVSFS_FILE_STATE_OPEN, PEM_KVSFS_TO_NFS);
```

Comment:
KVSFS tags!

Reply:
which is convenient function tag to use from handle.c?

Reply:
Same as previous

Reply:
fixed